packages/wallets/src/features/cfd/screens/Jurisdiction/JurisdictionCard/JurisdictionCard.tsx

```diff
@@ -97,7 +98,9 @@ const JurisdictionCard: React.FC<TJurisdictionCardProps> = ({ isSelected, jurisd
                                                     row.titleIndicators?.displayTextSkinColor || ''
                                                 }`}
                                             >
-                                                {row.titleIndicators?.displayText}
+                                                <WalletText color='white' size='xs' weight='bold'>
+                                                    {row.titleIndicators?.displayText}
```

Comment:
@adrienne-deriv just a nitpick, you can ignore it as well :) You can delete `?` after `titleIndicators` because you already checked this condition above. In my opinion we don't need to add extra checks if we can avoid it

Reply:
Nice catch, I've updated this as well: https://github.com/binary-com/deriv-app/pull/10739/commits/44e4875736230c4d995a07f54e4e6e432994d900